fdbserver/ClusterController.actor.cpp

```diff
@@ -3959,7 +3951,15 @@ ACTOR Future<Void> timeKeeper(ClusterControllerData* self) {
 				Version v = tr->getReadVersion().get();
 				int64_t currentTime = (int64_t)now();
 				versionMap.set(tr, currentTime, v);
+				if (!g_network->isSimulated()) {
+					// This is done to provide an arbitrary logged transaction every ~10s.
+					// FIXME: replace or augment this with logging on the proxy which tracks
+					//       how long it is taking to hear responses from each other component.
 
+					UID debugID = deterministicRandom()->randomUniqueID();
+					TraceEvent("TimeKeeperCommit", debugID).detail("Version", v).detail("Time", currentTime).log();
```

Comment:
The time field already exists, so you can't use it by the same name again. Just to show you what I mean, your PR description includes an event that looks like this:

```trace.0.0.0.0.0.1629275119.mlVgdn.0.1.xml:<Event Severity="10" Time="15.409136" DateTime="2021-08-18T08:25:19Z" Type="TimeKeeperCommit" Machine="2.0.1.1:1" ID="5f132f639a4ad17b" Version="407228" Timestamp="15" ThreadID="9788943724220871241" LogGroup="default" Roles="CC,CP,DD,GP,MS,RK,RV,SS,TL" />```

In particular, it has `Time="15.409136" ... Timestamp="15"`. This data comes from the same clock, so it's just going to be a floating point and integer representation of the same data.

Reply:
Although I suppose technically if someone were to disable the `now` based trace clock (a non-default option), they could be different. There's not really harm in having both fields here, so if it provides you with some value to have both I don't object to that.

Reply:
Just about to remove the Time field. Using the Time does make the field a little bit overloaded. It serves the purpose of debugging, though.